test/conformance/api/shared/util.go

```diff
@@ -56,8 +56,16 @@ func WaitForScaleToZero(t pkgTest.TLegacy, deploymentName string, clients *test.
 }
 
 // ValidateImageDigest validates the image digest.
-func ValidateImageDigest(imageName string, imageDigest string) (bool, error) {
-	ref, err := name.ParseReference(pkgTest.ImagePath(imageName))
+func ValidateImageDigest(imageName string, imageDigest string, multicontainer ...bool) (bool, error) {
```

Comment:
can you comment what the new parameter does.

Reply:
added new parameter in order to differentiate between single and multiple containers because `imageName` for multicontainers has been added to `pkgTest.ImagePath` in `service_test.go` testcase and inside this function again adding to `pkgTest.ImagePath` was creating problem.

But in order to avoid confusion now i reverted back changes to original way and handled this scenario as part of testcase.

So now there won't be any extra parameter.